pkg/sql/plan/function/builtin/multi/uuid_test.go

```diff
@@ -34,6 +34,7 @@ func TestUUID(t *testing.T) {
 	}
 
 	uuids := vector.GetStrVectorValues(res)
+	//uuids := vector.GetFixedVectorValues[types.Uuid](res)
```

Comment:
Are you sure?   Either use Fixed, or, remove this line.   This is confusing.

Reply:
I know this problem because at that time, there were two implementations of the return value type of the uuid() function: UUID type or string. At present, it is determined that the return value type of the uuid() function is string. Now the commented code is no longer needed. I delete it